src/ripple/app/ledger/Ledger.h

```diff
@@ -545,6 +545,25 @@ boost::optional<uint256>
 hashOfSeq (Ledger& ledger, LedgerIndex seq,
     SLECache& cache, beast::Journal journal);
 
+/** Find a ledger index from which we could easily get the requested ledger
```

Comment:
I think we need a name for this concept - otherwise as you think about the code, you have to keep the concept "an index from which we could easily get a specific ledger" in mind.

You're basically using "Candidate" as this name - but that just doesn't convey it for me.

What is it?  It's something like an index or a key, but it isn't.  It's like a post it note, or a bookmark, or the start of a chapter.

think think.

OK, I think there are _two_ concepts here:

1. being `0 mod 256`.
2. containing the index of the desired ledger.

We use these ideas a lot in the code - we should authoritatively name them, we use various names for this.

1. `0 mod 256` - I'd call that a "coset leader" or a "page leader" or a "segment leader"...
2. "an index to a ledger containing the hash of the desired ledger" - what about the verb "holds"?

So now we're trying to find a "page leader" ledger that "holds" the requested ledger.  Better...?

I welcome any choice of name you like.  The key part is naming these key concepts.